### PR TITLE
Extra quality metrics

### PR DESCRIPTION
**Pull request by gmurro**
Add extra quality metrics: BERTScore, Sentence BERT and Meteor.

A better default model has been setted for BERTScore (as described in this [sheet](https://docs.google.com/spreadsheets/d/1RKOVpselB98Nnh_EOC4A2BYn8_201tmPODpNWu4w7xI/edit#gid=0))
How to use them?

```
attack_results = attacker.attack_dataset()

sbert_stats = SBERTMetric().calculate(attack_results)
bert_score_stats = BERTScoreMetric().calculate(attack_results)
meteor_stats = MeteorMetric().calculate(attack_results)
```

### CODE REVIEW

**Review by jxmorris12 — CHANGES_REQUESTED** [review]
Code looks great, just one question about restoring defaults for BertScore. As long as you resolve that and tests pass, we should be good to merge.

File: textattack/constraints/semantics/bert_score.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ class BERTScore(Constraint):
     def __init__(
         self,
         min_bert_score,
-        model_name="bert-base-uncased",
-        num_layers=None,
+        model_name="microsoft/deberta-large-mnli",
```
[the remark this answers is not in the document]
Comment (gmurro) [reply textattack/constraints/semantics/bert_score.py:43]: Yeah, you are right! We can leave `bert-base-uncased` as default

[the remark this answers is not in the document]
Comment (jxmorris12) [reply textattack/constraints/semantics/bert_score.py:43]: can you fix and update the pr?

[the remark this answers is not in the document]
Comment (gmurro) [reply textattack/constraints/semantics/bert_score.py]: I just did, is there something else that I should do to merge the code?

**Review by jxmorris12 — APPROVED** [review]

**Pull request merged**